pages/operations/cost-management.md

```diff
@@ -30,10 +30,10 @@ As with any operations that involve taking down resources, you will likely encou
 
 ### Node Sizing at Scale
 
-If you deploy a lot of applications and resources, you may start seeing cost go up faster than expected due to the fixed cost of installing Kubernetes and its services on every node.
+If you deploy a lot of applications and resources, you may start seeing cost go up faster than expected due to the fixed cost of installing Kubernetes and its services on every node.  There's effectively a fixed kubernetes tax per node that varies per cloud provider, but if a small node is used, it will be a higher percentage of your usage than a large node, meaning it is more evident at large scale with small node sizes.
```

Comment:
```suggestion
If you deploy a lot of applications and resources, you may start seeing cost go up faster than expected due to the fixed cost of installing Kubernetes and its services on every node.  This fixed Kubernetes tax per node varies based on your cloud provider. Essentially, if a small node is used, it will be a higher percentage of your usage than a large node, making the cost differences more evident at large scale with small node sizes.
```

Reply:
What does the comment about usage mean exactly? Does it mean that a higher percentage of the node will be utilized?